src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -223,7 +223,8 @@ namespace OpenRCT2::Audio
             {
                 auto channel = *it;
                 int32_t group = channel->GetGroup();
-                if (group != MIXER_GROUP_SOUND || gConfigSound.sound_enabled)
+                if ((group != MIXER_GROUP_SOUND || gConfigSound.sound_enabled) && gConfigSound.master_sound_enabled
+                    && gConfigSound.master_volume != 0)
```

Comment:
Having master_volume == 0 is essentially identical to gConfigSound.master_sound_enabled == false which seems a little silly to me to have two options, I'm not requesting changes just stating the condition here. We should perhaps ditch the master volume option and simply check if its not 0 at some point.

Reply:
I disagree. It's nicer to be able to mute the volume and have it return to the old level when unmuting. 

Among other things, it would mean that clicking the Mute button in the toolbar, then unclicking it, would return the volume to full blast.

Reply:
Yes, but how often is one toggling it on and off? But I get your point for convenience reasons.